basicdataset/src/test/resources/mlrepo/dataset/nlp/ai/djl/basicdataset/WikiText-2/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Please use lower case folder name:

basicdataset/src/test/resources/mlrepo/dataset/nlp/ai/djl/basicdataset/wikitext-2/metadata.json

Reply:
> 

Linux file name is case sensitive. If you are on mac/Windows, you won't be able to rename the folder to lower case. You might need to remove WikiText-2 folder complete and recreate the folder with lower case.

Reply:
Thank you, I have changed it.